honeybee_radiance_command/getinfo.py

```diff
@@ -30,13 +30,25 @@ class Getinfo(Command):
         * input
     """
 
-    __slots__ = ('_input')
+    __slots__ = ('_input', '_header')
 
-    def __init__(self, options=None, output=None, input=None):
+    def __init__(self, options=None, output=None, input=None, header=None):
```

Comment:
Should we set the default value to `False`? Would a keyword like `remove_header` or `keep_header` be more explicit?

Reply:
We should add an explanation to docstrings